test/src/_assets/scss/components/_c-test-case.scss

```diff
@@ -109,13 +109,12 @@ $c-test-case-test-subject-height: 300;
 
     // Inner grid item including label
     .l-grid__item > div {
-        align-items: center;
         background-color: $g-color-test-case-subject-background;
-        display: flex;
         font-family: $shell-g-font-family-monospace;
         font-weight: normal;
-        height: $c-test-case-test-subject-height * 1px;
+        padding: 1rem;
```

Comment:
Shame we've lost the fixed heights (and `align-items: center;`) as I had them for demo purposes, as the grid items look pants when they're the height of 1 line of text:

![grid shell s tests](https://cloud.githubusercontent.com/assets/1342772/14512262/d30ff8ec-0220-11e6-90e1-87a611c10294.png)

And now there's inconsistency with the alignment modifiers who are still getting their fixed heights: https://github.com/davislurve/shell/blob/master/test/src/_assets/scss/components/_c-test-case.scss#L128.